src/ZlibSharp/ZlibSharp.csproj

```diff
@@ -1 +1,6 @@
-<Project Sdk="Microsoft.NET.Sdk" />
+<Project Sdk="Microsoft.NET.Sdk">
+    <ItemGroup>

```

Comment:
The package should go in to Directory.Packages.props.

Reply:
Oops, forgot to remove that